src/payment/strategies/mollie/mollie-initialize-options.ts

```diff
@@ -1,7 +1,65 @@
+/**
+ * A set of options that are required to initialize the Mollie payment method.
+ *
+ * Once Mollie payment is initialized, credit card form fields are provided by the
+ * payment provider as IFrames, these will be inserted into the current page. These
```

Comment:
Is the iframe inserted in `containerId` ?

Reply:
Nope

Reply:
ContainerId is for the container that will have the DIVs that will contain the iframe for each of the 4 Mollie Components, 
and i need it because i have to create and mount each of this Mollie Components every time the user selects to "add a new card" when having vaulted instruments, when they select a vaulted instrument, ContainerId element has display property as none, but when they select to add a new card it change to 'block' and that when i want to create and mount the MollieComponents and thats what i use containerId for